okhttp/src/jvmMain/kotlin/okhttp3/internal/publicsuffix/PublicSuffixDatabase.kt

```diff
@@ -237,7 +242,7 @@ class PublicSuffixDatabase {
 
   companion object {
     @JvmField
-    val PUBLIC_SUFFIX_RESOURCE = "${PublicSuffixDatabase::class.java.simpleName}.gz"
+    val PUBLIC_SUFFIX_RESOURCE = "/okhttp3/internal/publicsuffix/${PublicSuffixDatabase::class.java.simpleName}.gz".toPath()
```

Comment:
@swankjesse not 100% sure this is proguard safe, ideas?

I'll test on a snapshot after it lands.